polynote-frontend/polynote/ui/component/notebook.ts

```diff
@@ -180,6 +180,7 @@ export class NotebookUI extends UIMessageTarget {
                 removed.forEach(id => { this.removePresenceSelection(id); delete this.otherUsers[id] });
             })
             .when(messages.PresenceSelection, (id, cellId, range) => this.setPresenceSelection(id, cellId, range))
+            .when(messages.KernelError, (err) => console.log(err)) // TODO: need a general UI treatment for kernel-level errors
```

Comment:
Now kernel errors make it all the way to the client, but I'm not sure of the best way to present them. Modal dialog? Punted that decision.

Reply:
Maybe we could have a central log / error output location (e.g., in the About modal or somewhere else) where we can put the server/kernel stdout (could address https://github.com/polynote/polynote/issues/431).

Reply:
I don't really want to just crap out the logs in the UI. This could be a disaster in my view. But we should have some consistent way to surface errors which aren't from cell code. An example is when a notebook fails to load – we currently handle that specially, but that treatment doesn't really work for other errors.

Reply:
We don't have to firehose them over to the UI - but what do you think about a page that shows general errors and also lets you page through the logs?

Reply:
It'd be pretty useful to be able to see the logs from the UI, like when debugging an error on someone else's machine. It would also help to contextualize the errors that we want to surface. 

Anyways, we can talk about this some other time, not relevant to this PR for sure.